samples/snippets/cloud-client/src/main/java/recaptcha/account_defender/AccountDefenderAssessment.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package account_defender;
+
+// [START recaptcha_enterprise_account_defender_assessment]
+
+import com.google.cloud.recaptchaenterprise.v1.RecaptchaEnterpriseServiceClient;
+import com.google.protobuf.ByteString;
+import com.google.recaptchaenterprise.v1.AccountDefenderAssessment.AccountDefenderLabel;
+import com.google.recaptchaenterprise.v1.Assessment;
+import com.google.recaptchaenterprise.v1.CreateAssessmentRequest;
+import com.google.recaptchaenterprise.v1.Event;
+import com.google.recaptchaenterprise.v1.ProjectName;
+import com.google.recaptchaenterprise.v1.RiskAnalysis.ClassificationReason;
+import com.google.recaptchaenterprise.v1.TokenProperties;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+import java.util.UUID;
+
+public class AccountDefenderAssessment {
+
+  public static void main(String[] args) throws IOException, NoSuchAlgorithmException {
+    // TODO(developer): Replace these variables before running the sample.
+    // projectId: Google Cloud Project ID
+    String projectId = "project-id";
+
+    // recaptchaSiteKey: Site key obtained by registering a domain/app to use recaptcha
+    //    * services.
+    String recaptchaSiteKey = "recaptcha-site-key";
+
+    // token: The token obtained from the client on passing the recaptchaSiteKey.
+    String token = "recaptcha-token";
+
+    // recaptchaAction: Action name corresponding to the token.
+    String recaptchaAction = "recaptcha-action";
+
+    // Unique id of the customer (id can be email, customer id, etc.,).
+    String uniqueCustomerId = "default" + UUID.randomUUID().toString().split("-")[0];
+
+    // Hash the unique customer id using HMAC SHA-256.
+    MessageDigest digest = MessageDigest.getInstance("SHA-256");
+    byte[] hashBytes = digest.digest(uniqueCustomerId.getBytes(StandardCharsets.UTF_8));
+    ByteString hashedAccountId = ByteString.copyFrom(hashBytes);
+
+    accountDefenderAssessment(projectId, recaptchaSiteKey, token, recaptchaAction, hashedAccountId);
+  }
+
+  /**
+   * This assessment detects account takeovers. Input -> Pass in the hashed account id. Result ->
```

Comment:
What is "account takeover"? If this is explained in a document, can you reference it?

Reply:
Done, included the docs link in the comments.